### PR TITLE
Use go telegram bot api

### PR DESCRIPTION
**Pull request by DmitryRomanov**
- use tgbotapi
- dont use proxy
- install ca-certificates in balerter image

### CODE REVIEW

File: internal/alert/provider/telegram/api/message.go [hunk 1]
```diff
@@ -11,6 +19,36 @@ type PhotoMessage struct {
 	Caption string `json:"caption,omitempty"`
 }
 
+func (message *PhotoMessage) getPhotoFile() (fileName string, err error) {
+	tokens := strings.Split(message.Photo, "/")
+
+	_ = os.Mkdir("tmp", os.ModePerm)
```
Comment (negasus) [inline internal/alert/provider/telegram/api/message.go:25]: Maybe we should use ioutil.TempFile() ?

**Comment by negasus** [comment]
HI! Thanks for your PR!
Can you explain reasons for remote proxy support? Maybe we should use `tgbotapi.NewBotAPIWithClient()`?

File: internal/alert/provider/telegram/api/message.go (hunk 1 above)
Comment (DmitryRomanov) [reply internal/alert/provider/telegram/api/message.go]: > Maybe we should use ioutil.TempFile() ?

ok

**Comment by DmitryRomanov** [comment]
> HI! Thanks for your PR!
> Can you explain reasons for remote proxy support? Maybe we should use `tgbotapi.NewBotAPIWithClient()`?

I removed the connection through a proxy, cause that it is not available in other providers.

Also, i think, there is no need for proxying telegram now

**Comment by negasus** [comment]
> cause that it is not available in other providers.

I think, this make sense) thanks

File: internal/alert/provider/telegram/api/message.go [hunk 2]
```diff
@@ -11,6 +19,30 @@ type PhotoMessage struct {
 	Caption string `json:"caption,omitempty"`
 }
 
+func (message *PhotoMessage) getPhotoFile() (fileName string, err error) {
```
[the remark this answers is not in the document]
Comment (DmitryRomanov) [reply internal/alert/provider/telegram/api/message.go:22]: Yes, it's mistake. I've fixed.

File: internal/alert/provider/telegram/api/send.go [hunk 3]
```diff
@@ -1,43 +1,27 @@
...
-	res, err := api.httpClient.Do(req)
-	if err != nil {
-		return fmt.Errorf("error send request, %w", err)
-	}
-
-	res.Body.Close()
-
-	return nil
+func (api *API) sendMessage(message tgbotapi.Chattable) {
+	api.api.Send(message)
```
Comment (negasus) [inline internal/alert/provider/telegram/api/send.go:26]: We should check an error

**Comment by negasus** [comment]
I still don't understand how to use this library.
For example, now we don't need to download a photo file. With the new library, we have to do this.
What improvements will be gained by using the new library in general?

**Comment by DmitryRomanov** [comment]
> I still confused for use this library.
> For example, now we don't need to download a photo file. With the new library, we have to do this.
> What improvements will be gained by using the new library in general?



> I still confused for use this library.
> For example, now we don't need to download a photo file. With the new library, we have to do this.
> What improvements will be gained by using the new library in general?

**Comment by DmitryRomanov** [comment]
> I still confused for use this library.
> For example, now we don't need to download a photo file. With the new library, we have to do this.
> What improvements will be gained by using the new library in general?

Because balerter already uses telegram-bot-api in tgtool

File: internal/alert/provider/telegram/api/send.go (hunk 3 above)
Comment (DmitryRomanov) [reply internal/alert/provider/telegram/api/send.go]: fixed

**Pull request closed without merging**